vnext/stubs/glog/logging.h

```diff
@@ -69,6 +69,15 @@ inline std::ostream &GetNullLog() noexcept {
 }
 
 #define LOG(b) GlogStub::GetNullLog()
+
+typedef int LogSeverity;
+inline void FlushLogFiles(LogSeverity min_severity) {
+  LOG("");
```

Comment:
You can probably just no-op here.  Rather than including the LOG call.

Reply:
Ok

Reply:
@acoates-ms Is this line below ok?
#define google GlogStub
I didn't know how else to do it..